SimTracker/TrackerHitAssociation/python/tpClusterProducer_cfi.py

```diff
@@ -20,9 +20,6 @@
 )
 
 
-from SimTracker.TrackerHitAssociation.tpClusterProducerHeterogeneousDefault_cfi import tpClusterProducerHeterogeneousDefault as _tpClusterProducerHeterogeneous
-tpClusterProducerHeterogeneous = _tpClusterProducerHeterogeneous.clone()
-
-from SimTracker.TrackerHitAssociation.tpClusterHeterogeneousConverter_cfi import tpClusterHeterogeneousConverter as _tpHeterogeneousConverter
-tpClusterProducerConverter = _tpHeterogeneousConverter.clone()
+from SimTracker.TrackerHitAssociation.tpClusterProducerCUDADefault_cfi import tpClusterProducerCUDADefault as _tpClusterProducerCUDA
+tpClusterProducerCUDA = _tpClusterProducerCUDA.clone()
```

Comment:
Just to note that in principle `tpClusterProducerCUDA` should go in its own `cfi` file, or this `cfi` file should be changed to `cff`, but fine for now.